py/tests/trans-test.py

```diff
@@ -162,7 +160,7 @@ def test_am5_trans_src():
 
     plane = planes[1]
     fb = fbs[1]
-    z = 3
+    z = 1
```

Comment:
This breaks it for DSS5

Reply:
Ok. I'll add a separate test for DSS7, when have applied your comment to the current transparency key implementation.

Reply:
Thinking about this, I think it's about the number of planes. z has to be the num_planes - 1. So possibly we could do with a single test, although... Not sure how reliable it is in all cases.